LearningJourney/Modules/Features/JLibrary/JLibraryTests/Sources/Data/LibraryRepositoryTests.swift

```diff
@@ -0,0 +1,265 @@
+import XCTest
+import CoreNetworking
+@testable import JLibrary
+
+
+final class LibraryRepositoryTests: XCTestCase {
+    // MARK: - Properties
+
+    private var libraryRemoteServiceStub = LibraryRemoteServiceStub()
+    private var libraryParsingStub = LibraryParsingStub()
```

Comment:
These can be constants
```suggestion
    private let libraryRemoteServiceStub = LibraryRemoteServiceStub()
    private let libraryParsingStub = LibraryParsingStub()
```